src/features/dashboard/components/ApiTokenTable/index.tsx

```diff
@@ -3,19 +3,29 @@ import Spinner from '@site/src/components/Spinner';
 import styles from './api-table.module.scss';
 import useApiToken from '@site/src/hooks/useApiToken';
 import { Column } from 'react-table';
-import { TTokenType } from '@site/src/types';
 import ApiTokenCell from './table.token.cell';
 import ApiLastUsedCell from './table.lastused.cell';
+import { TTokenType } from '@site/src/types';
 import ScopesCell from '../Table/scopes.cell';
+import TokenActionsCell from './delete.token.cell';
+import AccountTypeCell from './account.type.cell';
 import Table from '../Table';
+import { Button, Heading, Text } from '@deriv-com/quill-ui';
+import { LabelPairedCirclePlusMdRegularIcon } from '@deriv/quill-icons';
+import ResponsiveTable from './responsive-table';
+import useDeviceType from '@site/src/hooks/useDeviceType';
 
 export type TTokenColumn = Column<TTokenType>;
 
-const tableColumns: TTokenColumn[] = [
+const tokenTableColumns = (): TTokenColumn[] => [
```

Comment:
I think we don't need to make it a function. The array should be fine to pass directly to the table.

Reply:
Yes @shafin-deriv , correct. Have changed this.